app/scripts/components/common/chart/constant.js

```diff
@@ -1,4 +1,5 @@
-export const chartHeight = '32rem';
+export const chartMaxHeight = 500;
```

Comment:
I had to use number instead of rem here because rechart only accepts number or % for maxHeight prop.